services/web/server/src/simcore_service_webserver/login/storage.py

```diff
@@ -48,11 +48,14 @@ def __init__(
     # CRUD user
     #
 
-    async def get_user(self, with_data) -> asyncpg.Record:
+    async def get_user(self, with_data) -> dict | None:
         async with self.pool.acquire() as conn:
-            return await _sql.find_one(conn, self.user_tbl, with_data)
+            result: asyncpg.Record | None = await _sql.find_one(
```

Comment:
when you put a type like this, you actually force it to be so. normally that is not needed. and both pylance and mypy shall know it is asyncpg.Record | None.

Reply:
Also you are now changing the syntax of that function. what happens if the user is None? is it handled?